<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZWebDownloadItem</class>
 <widget class="QWidget" name="ZWebDownloadItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="fileIcon">
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>48</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>图标</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="rightMargin">
        <number>15</number>
       </property>
       <item>
        <widget class="SqueezeLabel" name="fileNameLabel">
         <property name="text">
          <string>aaa</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="tryAgainButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{border-image: url(:/new/imageFile/ZWebViewBrowserImage/icon_yyzx_cxxz.png);}
QPushButton:hover{border-image: url(:/new/imageFile/ZWebViewBrowserImage/icon_yyzx_cxxz_sel.png);}
QPushButton:pressed{border-image: url(:/new/imageFile/ZWebViewBrowserImage/icon_yyzx_cxxz_sel.png);};</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="continuePushButton">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">
            QPushButton{border-image: url(:/new/imageFile/ZWebViewBrowserImage/icon_ks.png);}
            QPushButton:hover{border-image: url(:/new/imageFile/ZWebViewBrowserImage/icon_ks_sel.png);}
            QPushButton:pressed{border-image: url(:/new/imageFile/ZWebViewBrowserImage/icon_ks_sel.png);};</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="suspendButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">
            QPushButton{border-image: url(:/new/imageFile/ZWebViewBrowserImage/icon_zt.png);}
            QPushButton:hover{border-image: url(:/new/imageFile/ZWebViewBrowserImage/icon_zt_sel.png);}
            QPushButton:pressed{border-image: url(:/new/imageFile/ZWebViewBrowserImage/icon_zt_sel.png);};</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="openButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">
            QPushButton{border-image: url(:/new/imageFile/ZWebViewBrowserImage/icon_wj_nor.png);}
            QPushButton:hover{border-image: url(:/new/imageFile/ZWebViewBrowserImage/icon_wj_sel.png);}
            QPushButton:pressed{border-image: url(:/new/imageFile/ZWebViewBrowserImage/icon_wj_sel.png);};</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="openDirButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">
            QPushButton{border-image: url(:/new/imageFile/ZWebViewBrowserImage/icon_wjj_nor.png);}
            QPushButton:hover{border-image: url(:/new/imageFile/ZWebViewBrowserImage/icon_wjj_sel.png);}
            QPushButton:pressed{border-image: url(:/new/imageFile/ZWebViewBrowserImage/icon_wjj_sel.png);};</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">
            QPushButton{border-image: url(:/new/imageFile/ZWebViewBrowserImage/icon_del_nor.png);}
            QPushButton:hover{border-image: url(:/new/imageFile/ZWebViewBrowserImage/icon_del_sel.png);}
            QPushButton:pressed{border-image: url(:/new/imageFile/ZWebViewBrowserImage/icon_del_sel.png);};</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="SqueezeLabel" name="downloadInfoLabel">
       <property name="text">
        <string>bbb</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>1</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(113, 113, 113);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SqueezeLabel</class>
   <extends>QLabel</extends>
   <header location="global">ZWebViewBrowser/squeezelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
